### PR TITLE
Commit

### PR DESCRIPTION
**Pull request by DMITRYSHVED**

### CODE REVIEW

File: src/MySql_DZ_File.txt [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ insert into author(name,surname) values('dima','shved');
+ insert into book(title,pages,author_id) values ('asdf','123',1);
+ insert into author(name,surname) values('kirill','asdfg');
+ insert into book(title,pages,author_id) values ('lkjhg','456',2);
+ insert into author(name,surname) values('nbvc','wqertgyh');
+ insert into book(title,pages,author_id) values ('mnbvcxz','654',4);
+
+ select * from author join book on author.id = book.author_id;
+ select * from book where pages>300;
+ select * from author join book on author.id = book.author_id where pages > 300
```
Comment (DHrytsuk) [inline src/MySql_DZ_File.txt:22]: How about author name uniquness?
If we have more than 1 book, we will get more than 1 name (as a result of the select we need to get only author names/surnames)

File: src/MySql_DZ_File.txt [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+ insert into author(name,surname) values('dima','shved');
+ insert into book(title,pages,author_id) values ('asdf','123',1);
+ insert into author(name,surname) values('kirill','asdfg');
+ insert into book(title,pages,author_id) values ('lkjhg','456',2);
+ insert into author(name,surname) values('nbvc','wqertgyh');
+ insert into book(title,pages,author_id) values ('mnbvcxz','654',4);
+
+ select * from author join book on author.id = book.author_id;
+ select * from book where pages>300;
+ select * from author join book on author.id = book.author_id where pages > 300 group by name
```
Comment (DHrytsuk) [inline src/MySql_DZ_File.txt:22]: Does it really work? Have you checked it?

Comment (DMITRYSHVED) [reply src/MySql_DZ_File.txt:22]: Yes, sure

Comment (DHrytsuk) [reply src/MySql_DZ_File.txt:22]: It's strange)
but you should use DISTINCT instead

Comment (DMITRYSHVED) [reply src/MySql_DZ_File.txt]: Done, and check the pattern homework by the way

**Pull request merged**